function(resolve, reject) {
    throw new Error("Error in creation");
}